hacks/isolateVersion/allSteps.sh

```diff
@@ -30,12 +30,12 @@ script_index=$1
 
 script_directory=$(cd "$(dirname "$0")" && pwd)
 
-if [[ "$script_index" == 1 || -z "$script_index" ]]; then
-  source $script_directory/1-reduceVersionManifest.sh
+if [[ "$script_index" == 2 || -z "$script_index" ]]; then
+  source $script_directory/1-deleteOtherVersions.sh
 fi
 
-if [[ "$script_index" == 2 || -z "$script_index" ]]; then
-  source $script_directory/2-deleteOtherVersions.sh
+if [[ "$script_index" == 1 || -z "$script_index" ]]; then
+  source $script_directory/2-reduceVersionManifest.sh
```

Comment:
Is the index here right? The should be mismatched with the source?

Reply:
I definitely screwed something up -- I'll fix!

Reply:
Just like I screwed up that second sentence 😆 Thank you for interpreting my gibberish.

Reply:
fixed in latest commit!